locking/gradle.lockfile

```diff
@@ -0,0 +1,70 @@
+# This is a Gradle generated file for dependency locking.
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** CVE-2022-22950   
**Check Name:** spring-expression: Denial of service via specially crafted SpEL expression   
**Severity:** MEDIUM   
**Fixed Version:** 5.2.20.RELEASE, 5.3.17   
**Description:** n Spring Framework versions 5.3.0 - 5.3.16 and older unsupported versions, it is possible for a user to provide a specially crafted SpEL expression that may cause a denial of service condition.  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/nvd/cve-2022-22950